ako-operator/helm/ako-operator/values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for ako-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+# AKO Operator image settings
+operatorImage:
+  repository: 10.79.172.11:5000/ako-operator
+  pullPolicy: IfNotPresent
+
+# AKO Controller image settings
+akoImage:
+  repository: "10.79.172.11:5000/avi-buildops/ako:2.1.1-5097"
+  pullPolicy: IfNotPresent
+
+### This section outlines the generic AKO controller settings
+AKOSettings:
+  logLevel: "INFO" #enum: INFO|DEBUG|WARN|ERROR
+  fullSyncFrequency: "1800" # This frequency controls how often AKO polls the Avi controller to update itself with cloud configurations.
+  apiServerPort: 8080 # Specify the port for the API server, default is set as 8080 // EmptyAllowed: false
+  deleteConfig: "false" # Has to be set to true in configmap if user wants to delete AKO created objects from AVI 
+  disableStaticRouteSync: "false" # If the POD networks are reachable from the Avi SE, set this knob to true.
+  clusterName: "k8s-cluster" # A unique identifier for the kubernetes cluster, that helps distinguish the objects for this cluster in the avi controller. // MUST-EDIT
+  cniPlugin: "calico" # Set the string if your CNI is calico or openshift. enum: calico|canal|flannel|openshift 
+  # namespaceSelector contains label key and value used for namespacemigration
+  # same label has to be present on namespace/s which needs migration/sync to AKO
+  namespaceSelector:
+    labelKey: ""
+    labelValue: ""
+
+### This section outlines the network settings for virtualservices. 
+NetworkSettings:
+  ## This list of network and cidrs are used in pool placement network for vcenter cloud.
+  ## Node Network details are not needed when in nodeport mode / static routes are disabled / non vcenter clouds.
+  nodeNetworkList: []
+  # nodeNetworkList:
+  #   - networkName: "network-name"
+  #     cidrs:
+  #       - 10.0.0.1/24
+  #       - 11.0.0.1/24
+  subnetIP: "10.10.0.0" # Subnet IP of the vip network
+  subnetPrefix: "16" # Subnet Prefix of the vip network
+  networkName: "vcd-ns" # Network Name of the vip network
```

Comment:
We can keep it empty

Reply:
Yeah, this and all the other fields in `NetworkSettings`.